### PR TITLE
Add documentation for caching Ruby versions in non Ruby projects

### PR DESCRIPTION
**Pull request by javigomez**
Hi! :)

I am adding a quick trick in documentation to save more than 3 minutes in builds that are non Ruby based but that use Ruby executions in it.

Please add your thoughts. I am not English native

Thanks

### CODE REVIEW

**Comment by plaindocs** [comment]
Hi @javigomez 

Thanks for that. In this case I'm not sure showing the `.travis.yaml` is helpful, we can probably just use the description of "For example, a NodeJS application that has a Ruby functional test suite". 

What do you think?

File: user/caching.md [hunk 1]
```diff
@@ -76,6 +76,32 @@ In the cases where this is not desirable, you can use specify the [arbitrary dir
...
+  - cucumber features/
+```
+
+For these cases installing a version of ruby with `rvm install 2.3.1` may take more than 3 minutes. For these cases you can cache the ruby installation.
+
+```yaml
+ cache:
+    directories:
+     - /home/travis/.rvm/
+```
```
Comment (javigomez) [inline user/caching.md]: @plaindocs do you mean removing lines from 99 to 103?

I am okay with it. But this are the lines that people will have to add to their YML to make it work, I think they are valuable. Actually, I feel they give more value than lines from 83 to 95. 

Still, I am not documentation expert. So feel free to modify it as you think is the best for TravisCI

Comment (plaindocs) [reply user/caching.md]: I'm talkng about 83-95 - thanks

**Comment by javigomez** [comment]
@plaindocs updated with your suggestion.

**Comment by plaindocs** [comment]
Thanks!

**Pull request merged**